src/calculateRentalCost.js

```diff
@@ -28,7 +28,13 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days < 3) {
+    return 40 * days;
```

Comment:
Duplicated in 3 places, mb, move to const

```suggestion
    return 40 * days;
```